core/templates/pages/skill-editor-page/skill-preview-tab/skill-preview-tab.component.ts

```diff
@@ -145,6 +148,20 @@ export class SkillPreviewTabComponent implements OnInit, OnDestroy {
       });
   }
 
+  isSupplementalCardNonempty(card: StateCard): boolean {
+    return !card.isInteractionInline();
+  }
```

Comment:
Also, looks like this method is only used in one place, should we just use it inline where it is required?

Reply:
@kevintab95 Fixed this !